.travis.yml

```diff
@@ -16,6 +16,8 @@ branches:
 
 install:
   - pip install $DJANGO
+  # Testing, normal installation has issues
+  - pip install django-ipware
```

Comment:
See: https://github.com/un33k/django-ipware/issues/57

This has been bothering me throughout the day,
It's most probably because a `.pyc` file made it to the release of django-ipware.
Hopefully, they will re-release/fix it soon! :smile:

Reply:
Fixed